### DISCUSSION TITLE
new font

### DISCUSSION THREAD

**Discussion by icarok99** [Goldenfreddy0703/Otaku#141] [General]
It would be possible to add a new source with dubbed and subtitled content in portuguese ( pt-br )?

https://animes.vision/

**Comment by Dantesinferno11** [comment]
I like the otaku addon, could be possible to add spanish sources like 
Jkanime.net 
animelatinohd.com

**Reply by Goldenfreddy0703** [reply]
go ahead and put them in #144 sense someone wanted to make an issue on it.

**Reply by Goldenfreddy0703** [reply]
Heyyyyy @Dantesinferno11 good news, we added AnimeLatinoHD for both dub and sub. Hope this helps you out.
Huge thanks to @Gujal00 for adding it!!!!

**Reply by Dantesinferno11** [reply]
Thank you very much, now I can watch anime dubbed into Spanish. you are the best

**Comment by Urbexed** [comment]
would it be possible to add the French sources : 

- https://franime.fr/

- https://anime-sama.fr/

- https://neko-sama.fr/

thank you in advance if it's ever done

**Reply by Goldenfreddy0703** [reply]
Put these sources in #144

You can also use aniwatch to watch anime in French, just make sure your kodi's region settings are set to French so that you can use aniwatch to watch in French.
